Source/IFCExporterUIOverride/Properties/AssemblyInfo.cs

```diff
@@ -70,5 +70,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("19.1.0.0")]

```

Comment:
Also please back out changes to AssemblyInfo files.

Reply:
revert AssemblyVersion